dhis2/dhis2-core
# fix: token search with non ASCII alphanumeric characters [DHIS2-10323]

### Summary
Token based searches would strip all non-ASCII alphanumeric characters effectively preventing all scripts with other or additional letters from working correctly. This PR changes the kept characters from `a-zA-Z0-9` to all unicode script letters as well as 0-9.  

### Automatic Testing
Added a test scenario that adds a OU with a Cyrillic name and searches for it using token search like the Maintenance app does.

### Manual Testing
* create a metadata object with name `Кириллица`
* make sure other objects of the same type exist, either because they are already in the database or create them
* open Maintenance app and search for `Кириллица`
* check only the created item is listed
* check that searching for other terms matches the expected set of objects
